sound/soc/sof/sof-priv.h

```diff
@@ -639,4 +640,9 @@ int sof_stream_pcm_close(struct snd_sof_dev *sdev,
 			 struct snd_pcm_substream *substream);
 
 int sof_machine_check(struct snd_sof_dev *sdev);
+
+static inline int sof_get_abi_major(struct snd_sof_dev *sdev)
```

Comment:
Can you change the commit subject also? ;)

Reply:
> Can you change the commit subject also? ;)

Yes, sure. Sorry forgot this one. Done.